### PR TITLE
Feature new login page

### PR DESCRIPTION
**Pull request by emilianogp23**
- Se corrige Bug de login page 
- Se agregregan nuevos estilos al  Boostrap 
- Se Elimina boton home de recuperar contraseña y se agrega esa accion en el logo

TAS 360 #29

### CODE REVIEW

File: Views/Acceso/Login.cshtml [hunk 1]
```diff
@@ -114,18 +40,6 @@
 
             <!-- Enlace para recuperar contraseña -->
             <div class="forgot-password">
-            <!-- Contraseña -->
```
Comment (julianMondragon) [inline Views/Acceso/Login.cshtml:117]: Why delete?

**Review by julianMondragon — COMMENTED** [review]
check it out on you form

File: Views/Acceso/Login.cshtml [hunk 2]
```diff
@@ -114,18 +40,6 @@
...
-                <label for="pass">Contraseña:</label>
-                <input class="form-control" type="password" id="pass" name="pass" required />
-            </div>
-            <!-- Botones de Ingresar y Home en la misma línea -->
-            <div class="button-container">
-                <input type="submit" class="btn btn-primary" value="Ingresar" />
-            </div>
-            <!-- Enlace para recuperar contraseña -->
-            <div class="forgot-password">
-
```
Comment (julianMondragon) [inline Views/Acceso/Login.cshtml:128]: to here, why delete this code?

File: Views/Acceso/Login.cshtml [hunk 3]
```diff
@@ -114,18 +40,6 @@
...
                 @if (exception != null)
                 {
                     <pre class="error-message">@exception</pre>
```
Comment (julianMondragon) [inline Views/Acceso/Login.cshtml:45]: Why delete this part of code ?

**Review by julianMondragon — APPROVED** [review]

**Review by DulceAPTS — APPROVED** [review]
👍

**Pull request merged**